cvat-ui/package.json

```diff
@@ -35,6 +35,9 @@
     "@types/resize-observer-browser": "^0.1.6",
     "antd": "~4.18.9",
     "copy-to-clipboard": "^3.3.1",
+    "cvat-canvas": "file:../cvat-canvas",
+    "cvat-canvas3d": "file:../cvat-canvas3d",
+    "cvat-core": "file:../cvat-core",
```

Comment:
We should not add these dependencies here. It leads to bugs with ``yarn``
They were deleted before to avoid it

Reply:
Hot reload does not work when change these packages

Reply:
I've updated the pr -  tried to use  yarn's `link` instead of `file`. Hot reload woks fine for me with that change. Could you try it?